deploy/helm/elastic-agent/Chart.yaml

```diff
@@ -5,3 +5,8 @@ kubeVersion: ">= 1.27.0-0"
 type: application
 appVersion: 9.0.0
 version: 9.0.0-beta
+dependencies:
+  - name: kube-state-metrics
+    version: "5.28.0"
```

Comment:
Are we going to add any automation to keep this in-sync with released versions or is this going to be manual?

Reply:
good point 👌 I have to investigate how we could employ renovate to do that 🙂 do you believe this could be a follow-up PR?

Reply:
absolutely, in a follow up

this is already large enough ;-)